site/content/docs/5.3/examples/download-app/download-app.css

```diff
@@ -74,6 +74,18 @@
   display: none;
 }
 
+@media (prefers-color-scheme: light) {
+  [data-bs-theme="auto"] .dark-show {
+    display: none;
+  }
+}
+
+/* @media (prefers-color-scheme: dark) {
```

Comment:
Fix is working fine without this part.
I just can't decide if we should include it or not as defensive CSS 🤔